src/ucp/api/ucp.h

```diff
@@ -770,9 +770,16 @@ typedef struct ucp_ep_params {
     ucp_err_handling_mode_t err_mode;
 
     /**
-     * Handler to process transport level failure.
+     * Error handler callback, will be called with @ref user_data as an argument
+     * if set.
      */
-    ucp_err_handler_t       err_handler;
+    ucp_err_handler_cb_t    err_handler_cb;
+
+    /**
+     * User data associated with an endpoint. See @ref ucp_stream_poll_ep_t and
+     * @ref err_handler_cb
```

Comment:
should the ref be to ucp_err_handler_cb_t, rather than err_handler_cb?

Reply:
No, I referenced to the field above. These 2 fields may work together in case of error handling.

Reply:
ok..thx for the clarification.
:+1: for doc/API